cli/src/test/java/org/openapitools/openapistylevalidator/cli/MainTest.java

```diff
@@ -0,0 +1,68 @@
+package org.openapitools.openapistylevalidator.cli;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Options;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.openapitools.openapistylevalidator.styleerror.StyleError;
+import org.opentest4j.MultipleFailuresError;
+
+import java.util.List;
+
+public class MainTest {
+    private static final DefaultParser parser = new DefaultParser();
+
+    @Test
+    void wihtoutOptionFile() throws Exception {
+        OptionManager optionManager = new OptionManager();
+        Options options = optionManager.getOptions();
+        CommandLine commandLine = parser.parse(options, new String[]{"-s", "src/test/resources/ping.yaml"});
+        List<StyleError> errorList = Main.validate(optionManager, commandLine);
+
+        defaultAssertions(errorList);
+    }
+
+    @Test
+    void wihtEmptyOptionFile() throws Exception {
```

Comment:
Following what I said, this test would be named:

`validateShouldReturnDefaultAssertionWithEmptyOptionFile`